plugins/draw-tools.js

```diff
@@ -377,11 +385,33 @@ window.plugin.drawTools.optCopy = function() {
     if (stockWarnings.noCircle) stockWarnTexts.push('Warning: Circles cannot be exported to stock intel');
     if (stockWarnings.noMarker) stockWarnTexts.push('Warning: Markers cannot be exported to stock intel');
     if (stockWarnings.unknown) stockWarnTexts.push('Warning: UNKNOWN ITEM TYPE');
-
+/*
```

Comment:
don't comment, remove, we have history with git

Reply:
removed